transcom/mymove
# B-18161 Update email customer submitted moves-INT

## [B-18161](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871813)

## Summary

Updating email template that is sent when the customer creates a move with any shipment type.

- Fixed a typo from "an" to a"
- Added a sentence "If you have requested a PPM, DO NOT start your PPM until your counselor has approved it in MilMove. You will receive an email when that is complete."
- Changed email signature to "USTRANSCOM MilMove Team"

### How to test

For Gov't Counseling email:
1. Create a new customer with your email and add any shipment to the move.
2. Once you have finished submitting, you should receive an email with a title: "Thank you for submitting your move details.

For GHC Counseling email:
1. Create a new customer with your email, Marine Corps, and NS Mayport, FL 32228 as the duty location (Might have to pick 32233 as zip for your address if it doesn't work).
2. Choose any shipment for the move and finish submitting.
3. Once you have finished submitting, you should receive an email with a title: "Thank you for submitting your move details.

## Screenshots after the update

Gov't Counseling:
<img width="1026" alt="image" src="https://github.com/transcom/mymove/assets/146856854/89dabdd4-8b5d-4536-89cd-2015b607e22f">

GHC Counseling:
<img width="1055" alt="image" src="https://github.com/transcom/mymove/assets/146856854/04793e9f-a042-4507-b912-56a59ab54356">
